src/common/models/webhook.go

```diff
@@ -0,0 +1,121 @@
+package models
+
+import (
+	"encoding/json"
+	"time"
+)
+
+const (
+	// WebhookPolicyTable is table name for webhook policies
+	WebhookPolicyTable = "webhook_policy"
+	// WebhookJobTable is table name for webhook job
+	WebhookJobTable = "webhook_job"
+)
+
+// WebhookPolicy is the model for a webhook policy.
+type WebhookPolicy struct {
+	ID           int64        `orm:"pk;auto;column(id)" json:"id"`
+	Name         string       `orm:"column(name)" json:"name"`
+	Description  string       `orm:"column(description)" json:"description"`
+	ProjectID    int64        `orm:"column(project_id)" json:"project_id"`
+	TargetsDB    string       `orm:"column(targets)"`
+	Targets      []HookTarget `orm:"-" json:"targets"`
+	HookTypesDB  string       `orm:"column(hook_types)"`
+	HookTypes    []string     `orm:"-" json:"hook_types"`
+	Creator      string       `orm:"column(creator)" json:"creator"`
+	CreationTime time.Time    `orm:"column(creation_time);auto_now_add" json:"creation_time"`
+	UpdateTime   time.Time    `orm:"column(update_time);auto_now_add" json:"update_time"`
+	Enabled      bool         `orm:"column(enabled)" json:"enabled"`
+}
+
+// TableName set table name for ORM.
+func (w *WebhookPolicy) TableName() string {
+	return WebhookPolicyTable
+}
+
+// ConvertToDBModel convert struct data in webhook policy to DB model data
+func (w *WebhookPolicy) ConvertToDBModel() error {
+	if len(w.Targets) != 0 {
+		targets, err := json.Marshal(w.Targets)
+		if err != nil {
+			return err
+		}
+		w.TargetsDB = string(targets)
+	}
+	if len(w.HookTypes) != 0 {
```

Comment:
The `orm:"prop"` cannot correctly encode/decode the string array, something like `[]string{"a","b"}`? Have you tried it?

Reply:
> The `orm:"prop"` cannot correctly encode/decode the string array, something like `[]string{"a","b"}`? Have you tried it?

beego orm tag only support basic data type in 1 table